app/lib/three_scale/api/responder.rb

```diff
@@ -84,7 +84,7 @@ def serializable
   end
 
   def ordered_relation(relation)
-    if System::Database.postgres? && relation.order_values.empty?
+    if relation.order_values.empty?
```

Comment:
Some context on this:

After we stopped using old Oracle Arel visitors the API response for application list (for example) were coming in a seemingly random order (which is kind of OK, as DB doesn't guarantee an order if `ORDER BY` is not set).

I believe API responses (for "list" calls) must have a predictable order, especially on endpoints with pagination, so I think ordering by ID makes total sense (as almost all IDs are auto-incremented).

This was also discussed earlier: https://github.com/3scale/porta/pull/376/files#r250571722

Reply:
Regardless of what should happen in general, the method name suggests that this should return ordered entries, so it makes sense.